### ISSUE TITLE
DontIncludeToNamespace does nothing when exporting enums

### ISSUE THREAD

**Issue by jonsa** [reinforced/Reinforced.Typings#27]
Using the fluent configuration.

```C#
public static void Configure(ConfigurationBuilder builder)
{
  builder.ExportAsEnum<MyNamespace.MyEnum>()
    .DontIncludeToNamespace();

  builder.ExportAsInterface<MyNamespace.MyModel>()
    .DontIncludeToNamespace();
}
```
results in
```typescript
declare module MyNamespace {
  export enum MyEnum { 
    Foo = 0, 
    Bar = 1, 
  }
}

interface IMyModel
{
}
```

**Comment by pavel-b-novikov** [comment]
Fixed in 1.30 branch. Please check out and run your test

**Comment by pavel-b-novikov** [comment]
Fixed in 1.3.0
